dali/ft/filecopy.cpp

```diff
@@ -3327,6 +3247,12 @@ void FileSprayer::updateTargetProperties()
             }
             curHistory->addPropTree("Origin",newRecord.getClear());
         }
+
+        int expireDays = options->getPropInt("@expireDays");
+        if (expireDays != -1)
+        {
+            curProps.setPropInt("@expireDays", expireDays);
+        }
```

Comment:
I can't see the difference between previous PR state and current (I assume squashed?)..
but I don't think this was here before and doesn't look related to this JIRA.